chart/fluentbit-operator/values.yaml

```diff
@@ -50,6 +50,16 @@ resources:
      cpu: 10m
      memory: 25Mi
 
+#Set a limit of memory that Tail plugin can use when appending data to the Engine.
```

Comment:
=>
```
# Set a limit of memory that Tail plugin can use when appending data to the Engine.
# If the limit is reach, it will be paused; when the data is flushed it resumes.
# If the inboud traffic is less than 2.4Mbps, setting memBufLimit to 5MB is enough
# If the inboud traffic is less than 4.0Mbps, setting memBufLimit to 10MB is enough
# If the inboud traffic is less than 13.64Mbps, setting memBufLimit to 50MB is enough
```